src/Microsoft.Framework.Runtime.Interfaces/IServiceProviderLocator.cs

```diff
@@ -5,8 +5,14 @@
 
 namespace Microsoft.Framework.Runtime.Infrastructure
 {
+    /// <summary>
+    /// The <see cref="IServiceProviderLocator"/> provides access to the current <see cref="IServiceProvider"/>.
+    /// </summary>
     public interface IServiceProviderLocator
     {
+        /// <summary>
+        /// Gets or sets the <see cref="IServiceProvider"/>.
+        /// </summary>
```

Comment:
<sarcasm>I was expecting a property of type `IServiceProvider` with get and set to do something else and not get or set the `IService` provider</sarcasm> :stuck_out_tongue: